spring-cloud-gcp-docs/src/main/asciidoc/security-iap.adoc

```diff
@@ -35,6 +35,23 @@ dependencies {
 }
 ----
 
+== Customizable Properties
+The following properties are available with `spring.cloud.gcp.security.iap` prefix:
```

Comment:
There should probably be a some language about not modifying these for production use and only using for testing.